secure_message/resources/messages.py

```diff
@@ -253,19 +249,18 @@ def _validate_request(request_data):
 
 
 class MessageCounter(Resource):
-
-    """Get a count of unread messages"""
-
+    """Get count of unread messages using v2 endpoint"""
     @staticmethod
     def get():
-        """Get count of unread messages"""
-        logger.info("Getting count of unread messages", user_uuid=g.user.user_uuid)
-        try:
-            if request.args.get('name').lower() == 'unread':
-                return jsonify(name=request.args['name'], total=Retriever().unread_message_count(g.user))
+        if request.args.get('label'):
+            name = str(request.args.get('label'))
+            survey = request.args.get('survey')
+            if name.lower() == 'unread':
+                message_service = Retriever()
```

Comment:
I can see this has been copied from the v2 functionality, but can we remove the badly named `message_service` variable and call Retriever() directly below?

Reply:
Done